propulsion-reactor/Program.fs

```diff
@@ -496,7 +525,8 @@ let build (args : Args.Arguments) =
 //#if (!changeFeedOnly)
     match args.SourceParams() with
     | Choice1Of2 (srcE, cosmos, spec) ->
-        let connectEs () = srcE.Connect(Log.Logger, Log.Logger, AppName, Equinox.EventStore.ConnectionStrategy.ClusterSingle Equinox.EventStore.NodePreference.PreferSlave)
+        let connectEs () = srcE.Connect(Log.Logger, Log.Logger, AppName, Equinox.EventStore.ConnectionStrategy.ClusterSingle Equinox.EventStore.NodePreference.Master)
+        let connectProjEs = srcE.ConnectProj(Log.Logger, Log.Logger, AppName, Equinox.EventStore.NodePreference.Master)
```

Comment:
could/should this be nice and PreferSlave?

Reply:
PreferSlave should work for Proj connection